common/script/ops/rebirth.js

```diff
@@ -5,13 +5,12 @@ import { MAX_LEVEL } from '../constants';
 import {
   NotAuthorized,
 } from '../libs/errors';
-import resetGear from '../fns/resetGear';
 import equip from './equip';
 
 const USERSTATSLIST = ['per', 'int', 'con', 'str', 'points', 'gp', 'exp', 'mp'];
 
 module.exports = function rebirth (user, tasks = [], req = {}, analytics) {
-  if (user.balance < 2 && user.stats.lvl < MAX_LEVEL) {
+  if (user.balance < 1.5 && user.stats.lvl < MAX_LEVEL) {
```

Comment:
Let's not worry about this right now. I can extract the data into a content object when the PR is ready to be merged.